src/components/base/InternalTextInput/InternalTextInput.stories.tsx

```diff
@@ -3,30 +3,16 @@ import { Meta, StoryObj } from "@storybook/react";
 
 import { InternalTextInput } from "./InternalTextInput";
 
-/**
- *
- * An unstyled input to be used as a basis for UI input components.
- * The following callbacks are available for tracking focus events:
- *
- *  ### onFocus
- * `(e: FocusEvent<HTMLInputElement>) => void;`
- *  ### onBlur
- * `(e: FocusEvent<HTMLInputElement>) => void;`
- *  ### onInitialFocus
- * `(e: FocusEvent<HTMLInputElement>) => void;`<br>
- *  occurs only when the input is focused for the first time
- *
- */
+import { sizeArgTypes } from "@/storybook-helpers/sizeStyleHelpers";
+import { spacingArgTypes } from "@/storybook-helpers/spacingStyleHelpers";
 
 const meta: Meta<typeof InternalTextInput> = {
   component: InternalTextInput,
   tags: ["autodocs"],
   title: "components/base/InternalTextInput",
-  argTypes: {},
-  parameters: {
-    controls: {
-      include: [],
```

Comment:
removing this field populates the story with all possible props. The designers complained that these made the stories unnavigable for them. We therefore adopt an approach of putting a curated list of props in the story - namely those props which can be meaningfully manipulated and are specific to that component.